Changelog.md

```diff
@@ -1,17 +1,19 @@
 # Changelog
 
 ## [unreleased]
-- Remove group name displayed attribute from assignment properties table (#5834)
```

Comment:
@pretendWhale I think you deleted this line altogether, was that intentional? It's more of an internal change I suppose, but I wanted to see what your thoughts were.